System.IO.FileSystem/Path.cs

```diff
@@ -139,6 +139,9 @@ public static string GetDirectoryName(string path)
 
                     var lastPathPostion = path.LastIndexOf(DirectorySeparatorChar);
 
+                    if (lastPathPostion == -1)
```

Comment:
Makes sense!

Please follow project code style and add open/close brackets to if clause.

Reply:
Sure, I have fixed my C# style about lacking of brackets.

However I read this document: https://docs.nanoframework.net/content/contributing/cs-coding-style.html and in the example file, perhaps in 18 line you do not use brackets for if clause. That is why I thought we can use short version of this syntax.

Anyway fixed :) Thank you for quick reply.

Reply:
You have a point... 
That example it's outdated. And we should have a config file for these rules anyway. 😞 

Thanks for the fix!

Reply:
+1 for config file rule
Thank you for cooperation.
Have a nice day :)